### ISSUE TITLE
Dead Link/Incorrect URL in

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4507]
|<img alt="Grubulous's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Grubulous reported|
|-|-|
|Trac ID|trac#4507|
|Type|defect|
|Created|2010-06-16 22:13:17Z|
|Branch|https://github.com/twisted/twisted/tree/broken-client-link-4507|

Under the **Receiving Responses** of the web client documentation, the link to the deliverBody method seems broken.

Link:
[web client](http://twistedmatrix.com/documents/current/web/howto/client.html)

Problem Link within page:
```
<code class="API"><a href="http://twistedmatrix.com/documents/10.0.0/api/twisted.web._newclient.Response.deliverBody.deliverBody.html" title="twisted.web._newclient.Response.deliverBody.deliverBody">deliverBody</a></code>
```

changing
'deliverBody.deliverBody.html'
to
'deliverBody.html' seems to work

<details><summary>Searchable metadata</summary>

```
trac-id__4507 4507
type__defect defect
reporter__Grubulous Grubulous
priority__normal normal
milestone__ 
branch__branches_broken_client_link_4507 branches/broken-client-link-4507
branch_author__jesstess jesstess
status__closed closed
resolution__fixed fixed
component__web web
keywords__ 
time__1276726397000000 1276726397000000
changetime__1276829870000000 1276829870000000
version__None None
owner__ 
cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess set owner to @jesstess|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_2"></a>|@jesstess commented|
|-|-|

(In [29314]) Branching to 'broken-client-link-4507'

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_3"></a>|@jesstess commented|
|-|-|

(In [29315]) Fix broken links to deliverBody and iweb.UNKNOWN_LENGTH.

refs [#4507](https://github.com/twisted/twisted/issues/4507)

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set owner to @jesstess|
|-|-|

We have a ".doc" news fragment type now which would be appropriate for this.  The actual change looks good.  Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_6"></a>|@jesstess commented|
|-|-|

(In [29322]) .misc ==> .doc

refs [#4507](https://github.com/twisted/twisted/issues/4507)

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_7"></a>|@jesstess set status to `closed`|
|-|-|

(In [29323]) Merge broken-client-link-4507

Author: jesstess
Reviewer: exarkun
Fixes: [#4507](https://github.com/twisted/twisted/issues/4507)

Fix broken links to deliverBody and iweb.UNKNOWN_LENGTH in
doc/web/howto/client.xhtml.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
